docs/development/setup_development.rst

```diff
@@ -98,13 +98,16 @@ Fork & Clone the Repository
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Now you are ready to get your own copy of the source code.
-Visit our repository_ fork it and clone it on you local machine:
-
+Visit our repository_ fork it and clone it on your local machine.
 
 .. code:: sh
 
    git clone git@github.com:<your_github_username>/securedrop.git
 
+.. note:: The ``develop`` branch is the primary branch for development. Code 
+   changes should be pushed to the ``develop`` git branch of our repository_.
```

Comment:
Technically, we don't want contributors to push to `develop` directly, but to open PRs, which are then merged by a maintainer.

I would simplify: 

> PRs should be opened against the ``develop`` branch of our repository, which is the primary branch used for development.

Or something to that effect.